update.sh

```diff
@@ -375,6 +408,107 @@ patch__compose__remove_invoice_send_email() {
     fi
 }
 
+patch__compose__remove_supervisord() {
+    if ! (is_updating_to_version "${UPDATING_TO}" "2007000" 0 1); then
+        return 1
+    fi
+
+    if cat -vt "${UCRM_PATH}/docker-compose.yml" | grep -Eq "  supervisord:";
+    then
+        echo "Your docker-compose contains obsolete section supervisord. Trying to remove."
+        sed -i -e '/supervisord/,/^  [^ ]/{//!d}' "${UCRM_PATH}/docker-compose.yml"
+        sed -i -e '/supervisord/d' "${UCRM_PATH}/docker-compose.yml"
+
+        return 0
+    else
+        return 1
+    fi
+}
+
+patch__compose__remove_sync_app() {
+    if ! (is_updating_to_version "${UPDATING_TO}" "2007000" 0 1); then
+        return 1
+    fi
+
+    if cat -vt "${UCRM_PATH}/docker-compose.yml" | grep -Eq "  sync_app:";
+    then
+        echo "Your docker-compose contains obsolete section sync_app. Trying to remove."
+        sed -i -e '/sync_app/,/^  [^ ]/{//!d}' "${UCRM_PATH}/docker-compose.yml"
+        sed -i -e '/sync_app/d' "${UCRM_PATH}/docker-compose.yml"
+
+        return 0
+    else
+        return 1
+    fi
+}
+
+patch__compose__remove_crm_search_devices_app() {
+    if ! (is_updating_to_version "${UPDATING_TO}" "2007000" 0 1); then
+        return 1
+    fi
+
+    if cat -vt "${UCRM_PATH}/docker-compose.yml" | grep -Eq "  crm_search_devices_app:";
+    then
+        echo "Your docker-compose contains obsolete section crm_search_devices_app. Trying to remove."
+        sed -i -e '/crm_search_devices_app/,/^  [^ ]/{//!d}' "${UCRM_PATH}/docker-compose.yml"
+        sed -i -e '/crm_search_devices_app/d' "${UCRM_PATH}/docker-compose.yml"
+
+        return 0
+    else
+        return 1
+    fi
+}
+
+patch__compose__remove_crm_netflow_app() {
+    if ! (is_updating_to_version "${UPDATING_TO}" "2007000" 0 1); then
+        return 1
+    fi
+
+    if cat -vt "${UCRM_PATH}/docker-compose.yml" | grep -Eq "  crm_netflow_app:";
+    then
+        echo "Your docker-compose contains obsolete section crm_netflow_app. Trying to remove."
+        sed -i -e '/crm_netflow_app/,/^  [^ ]/{//!d}' "${UCRM_PATH}/docker-compose.yml"
+        sed -i -e '/crm_netflow_app/d' "${UCRM_PATH}/docker-compose.yml"
+
+        return 0
+    else
+        return 1
+    fi
+}
+
+patch__compose__remove_crm_ping_app() {
+    if ! (is_updating_to_version "${UPDATING_TO}" "2007000" 0 1); then
+        return 1
+    fi
+
+    if cat -vt "${UCRM_PATH}/docker-compose.yml" | grep -Eq "  crm_ping_app:";
+    then
+        echo "Your docker-compose contains obsolete section crm_ping_app. Trying to remove."
+        sed -i -e '/crm_ping_app/,/^  [^ ]/{//!d}' "${UCRM_PATH}/docker-compose.yml"
+        sed -i -e '/crm_ping_app/d' "${UCRM_PATH}/docker-compose.yml"
+
+        return 0
+    else
+        return 1
+    fi
+}
+
+patch__compose__add_udp_to_web_app() {
+    if ! (is_updating_to_version "${UPDATING_TO}" "2007000" 0 1); then
+        return 1
+    fi
+
+    if ! cat -vt "${UCRM_PATH}/docker-compose.yml" | grep -Eq "      - 2055:2055/udp";
```

Comment:
this port can be manually customized by user, customization should be kept and only moved to web_app container

Reply:
I have no idea how to make that work.